service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -559,7 +603,7 @@ void tooManyRecordIds() throws Exception {
     String inputSourceAsString = "{ \"type\" : \"none\", \"record_attribute\" : null }";
     String request =
         requestTemplate.formatted(
-            workflowUrl,
+            methodVersionId,
```

Comment:
Turns out this test was actually going sideways but since we were checking for an error it didn't pick up on it until I added the GlobalExceptionHandler, which makes thrown errors handled more accurately (and doesn't seem to impact ResponseEntities with error status codes)